devtools/maven/src/main/java/io/quarkus/maven/DevMojo.java

```diff
@@ -225,9 +225,7 @@ public void execute() throws MojoFailureException, MojoExecutionException {
             }
             //we don't want to just copy every system property, as a lot of them are set by the JVM
             for (Map.Entry<Object, Object> i : System.getProperties().entrySet()) {
-                if (i.getKey().toString().startsWith("quarkus.")) {
-                    args.add("-D" + i.getKey() + "=" + i.getValue());
-                }
+                args.add("-D" + i.getKey() + "=" + i.getValue());
```

Comment:
Please see comment on line 226: not sure if this is a good idea?

But if we decide to apply the patch, the comment should be updated.